src/main/java/frc/robot/subsystems/elevator/SimulatedElevator.java

```diff
@@ -28,7 +28,7 @@ public ElevatorIOSim() {
   }
 
   @Override
-  public void updateInputs(ElevatorIOInputs inputs) {
+  public void updateInputs(ElevatorInterfaceInputs inputs) {
     elevatorSim.update(0.02);
```

Comment:
I know this wasn't the point of the pr and you didn't do this, but 0.02 should really be a constant. Maybe we make like a SimulationConstants class (if we don't already have them.)

Additionally, you could make the constant something like `SIMULATION_TIME_STEP = 1 / 50.0;` To make it very clear why that number is 0.02.

Reply:
the constant should already exist, we just don't use it everywhere